php/updateFunctions.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+require_once 'dbConn.php';
+
+$db = dbConn();
+
+
+
+    /*
+    * This Function Updates given fields into the database.
+    *
+    * @param the $db is to input the new instance of PDO database.
+    * @param the $title contains the value of title to be inputted.
```

Comment:
change the for string and swap content and title variables to read in order of input